src/app/WriteClient.cpp

```diff
@@ -144,13 +122,8 @@ CHIP_ERROR WriteClient::ProcessWriteResponseMessage(System::PacketBufferHandle &
     return err;
 }
 
-CHIP_ERROR WriteClient::PrepareAttribute(const AttributePathParams & attributePathParams)
+CHIP_ERROR WriteClient::PrepareAttributeIB(const ConcreteDataAttributePath & attributePathParams)
```

Comment:
This will do the wrong thing for group writes, no?

Reply:
I see, group write is not broken by a series of invalid values.